cds-snc/notification-admin
# Bugfix - Table styling on the csv "review before sending" page

# Summary | Résumé

Connected to https://github.com/cds-snc/notification-planning/issues/744

On the CSV review page w/ no errors, there were actually 2 tables which I have combined. On the error page there was only 1 table and all I had to do there was remove some problematic javascript that was getting called.

<img width="1353" alt="Screen Shot 2022-09-23 at 3 50 56 PM" src="https://user-images.githubusercontent.com/5498428/192046762-9f595574-684a-4226-8c50-42410a754c8d.png">
<img width="1352" alt="Screen Shot 2022-09-23 at 3 49 55 PM" src="https://user-images.githubusercontent.com/5498428/192046764-659cc1af-214b-4e4c-8189-10264fce80f8.png">

## Test instructions

Try doing a CSV send using a CSV with errors (bad phone number) and without. Make sure that the issue described in https://github.com/cds-snc/notification-planning/issues/744 has been fixed.
